bazel-mode.el

```diff
@@ -21,6 +21,35 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode-buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
+  :group 'bazel-mode)
+
+(defun bazel-mode-buildifier ()
+  "Format current buffer using buildifier."
+  (interactive "*")
+  (let ((temp-file (make-temp-file "buildifier" nil nil (buffer-string)))
+        (input-buffer (current-buffer))
+        (buildifier-buffer (get-buffer-create "*buildifier*")))
+    (with-current-buffer buildifier-buffer
+      (erase-buffer)
+      (let ((return-code
+             (process-file buildifier-cmd temp-file buildifier-buffer nil "-type=build")))
+        (unwind-protect
```

Comment:
this `unwind-protect` doesn't seem to have any unwind forms?

Reply:
Done. That's where the temp file cleanup now lives.